misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -117,12 +119,22 @@ class JettyService @Inject internal constructor(
     }
 
     // TODO(mmihic): Force security handler?
+
+    val handlers = HandlerCollection()
+
     val servletContextHandler = ServletContextHandler()
     servletContextHandler.addServlet(ServletHolder(webActionsServlet), "/*")
-    server.addManaged(servletContextHandler)
-    server.handler = servletContextHandler
+    handlers.addHandler(servletContextHandler)
+
+    val statisticsHandler = StatisticsHandler()
+    statisticsHandler.server = server
+    handlers.addHandler(statisticsHandler)
 
     server.stopAtShutdown = true
+    server.stopTimeout = 20_000
```

Comment:
20 seconds is arbitrary but good since kubernetes gives us 30 second

Reply:
Promote this to a code comment?

Reply:
Is this line the only behavior change?